app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def show; end
```

Comment:
- [ ] @jicamargo Great work on your controllers, however, the user controller should have both an index and show methods, It's rails best practice to have only one controller for a particular model. This will make your code easier to read and scale in the future.